src/gui/socketapi/socketapi.cpp

```diff
@@ -752,6 +759,11 @@ class GetOrCreatePublicLinkShare : public QObject
         _shareManager.fetchShares(_localFile);
     }
 
+    void setIsE2eEncryptedFolder(bool isE2eEncryptedFolder)
+    {
+        _isE2eEncryptedFolder = isE2eEncryptedFolder;
```

Comment:
@claucambra Sorry, but, what is the point? What are we winning with Q_PROPERTY in the case when we never use this class in QML or Designer, etc? Why complicate stuff and write more code? This `bool` is just a flag to alter the behavior of `GetOrCreatePublicLinkShare` and is not something that gets changed dynamically, I can change it to be in the constructor instead of using a setter if that improves.

Reply:
Since it is used here as a set and forget private value then yes, this would be even better :)